include/constants/battle_config.h

```diff
@@ -89,6 +90,7 @@
 #define B_ROOTED_GROUNDING          GEN_LATEST // In Gen4+, Ingrain causes the affected Pokémon to become grounded.
 #define B_METRONOME_MOVES           GEN_LATEST // This config will determine up to which generation will Metronome pull moves from.
 #define B_TELEPORT_BEHAVIOR         GEN_LATEST // In Gen7+, starting with Pokémon LGPE, Teleport allows the user to swap out with another party member.
+#define B_BEAT_UP_DMG               GEN_LATEST // In Gen5+, Beat Up uses a different formula to calculate the damage of each hit, and deals Dark-type damage.
```

Comment:
I'm thinking we should rename the define to reflect that it's not only the damage. `B_BEAT_UP` should be enough. Also, better add in that it announces mon names pre-Gen5